scripts/environment/bootstrap-ubuntu-20.04.sh

```diff
@@ -117,3 +117,24 @@ mv --force --verbose "$TEMP"/bin/protoc /usr/bin/protoc
 
 # Apt cleanup
 apt clean
+
+# Install mold, because the system linker wastes a bunch of time.
+TEMP=$(mktemp -d)
+MOLD_VERSION=1.2.0
+MOLD_TARGET=mold-${MOLD_VERSION}-x86_64-linux
+curl -fsSL "https://github.com/rui314/mold/releases/download/v${MOLD_VERSION}/${MOLD_TARGET}.tar.gz" \
+     --output "$TEMP/${MOLD_TARGET}.tar.gz"
+tar \
+    -xvf "${TEMP}/${MOLD_TARGET}.tar.gz" \
+    -C "${TEMP}"
+cp "${TEMP}/${MOLD_TARGET}/bin/mold" /usr/bin/mold
+
+# Set Cargo to use mold as its linker.
+CARGO_OVERRIDE_DIR="${HOME}/.cargo"
+mkdir -p "${CARGO_OVERRIDE_DIR}"
+
+CARGO_OVERRIDE_CONF="${CARGO_OVERRIDE_DIR}/config.toml"
+cat <<EOF >"${CARGO_OVERRIDE_CONF}"
+[target.x86_64-unknown-linux-gnu]
+rustflags = ["-C", "linker=clang", "-C", "link-arg=-fuse-ld=/usr/bin/mold"]
+EOF
```

Comment:
What do you think about setting `rustc-wrapper` instead to something that calls `mold -run "$@"`?

Reply:
Hmmm... I'm not against it, but out of curiosity: is there a benefit to running that way?

Reply:
I recall hitting errors using mold with `rustflags`, so no real benefit if this works for you. I'll be playing around with adding `sccache` to the check-component-features run, which needs the wrapper.